### PR TITLE
Change all http to https

### PR DESCRIPTION
**Pull request by findrakecil**
Since HTTPS is a standard and all links to FuelPHP site in this repo still `http`, I changed all `http` text to `https`

### CODE REVIEW

**Comment by WanWizard** [comment]
Please remove the URL changes. This is done for a purpose, it allows moving the sub-sites and/or changing the redirects without breaking the documentation of existing installations.

**Comment by WanWizard** [comment]
Comments still stand...

**Comment by findrakecil** [comment]
> Please remove the URL changes. This is done for a purpose, it allows moving the sub-sites and/or changing the redirects without breaking the documentation of existing installations.

Ah, yes.. I see.. Thank you for the explanation. 

I have removed the URL changes commit.

File: fuel/app/views/welcome/index.php [hunk 1]
```diff
@@ -50,24 +50,24 @@
 			</div>
 			<div class="col-md-4">
 				<h2>Learn</h2>
-				<p>The best way to learn FuelPHP is reading through the <a href="http://docs.fuelphp.com">Documentation</a>.</p>
-				<p>Another good resource is the <a href="http://fuelphp.com/forums">Forums</a>.  They are fairly active, and you can usually get a response quickly.</p>
+				<p>The best way to learn FuelPHP is reading through the <a href="https://docs.fuelphp.com">Documentation</a>.</p>
+				<p>Another good resource is the <a href="https://fuelphp.com/forums">Forums</a>.  They are fairly active, and you can usually get a response quickly.</p>
```
Comment (WanWizard) [inline fuel/app/views/welcome/index.php]: This one is still wrong?

Comment (findrakecil) [reply fuel/app/views/welcome/index.php]: Pardon me. Could you tell me which one is still wrong?

The URL I changed in the last commit back then was `https://docs.fuelphp.com` to `https://fuelphp.com/docs`. After I delete the commit, it changed back to `https://docs.fuelphp.com`

The origin URL was `http://docs.fuelphp.com`

Comment (WanWizard) [reply fuel/app/views/welcome/index.php]: It is not only docs, but also forums you changed. It should be `https://forums.fuelphp.com`.

Comment (findrakecil) [reply fuel/app/views/welcome/index.php]: Hmm.. I don't remember changed the forum URL 🤔

But, OK. I will change the forum URL. Thank you for pointing out the problem

Comment (WanWizard) [reply fuel/app/views/welcome/index.php]: Thanks, I'll merge when the change is applied.

**Comment by findrakecil** [comment]
Hello, @WanWizard I have changed the URL to FuelPHP forums

**Comment by WanWizard** [comment]
Thanks !

**Pull request merged**